app/src/main/kotlin/com/wire/android/navigation/NavigationManager.kt

```diff
@@ -27,16 +27,16 @@ import kotlinx.coroutines.flow.SharedFlow
 class NavigationManager {
 
     private val _navigateState = MutableSharedFlow<NavigationCommand?>()
-    private val _navigateBack = MutableSharedFlow<Map<String, Any>>()
+    private val _navigateBack = MutableSharedFlow<Unit>()
     var navigateState: SharedFlow<NavigationCommand?> = _navigateState
-    var navigateBack: SharedFlow<Map<String, Any>> = _navigateBack
+    var navigateBack: SharedFlow<Unit> = _navigateBack
```

Comment:
Just temporary change, whole manager to be removed soon after all screens are refactored